project/plugins.sbt

```diff
@@ -5,3 +5,5 @@ resolvers += Resolver.url("Typesafe snapshot repository", new java.net.URL("http
 libraryDependencies <+= (sbtVersion).apply { sv =>
```

Comment:
The couple lines before this look like a really old syntax and URL. How about:

    resolvers += "Typesafe repository" at "https://repo.typesafe.com/typesafe/releases/"